fastly/platform_tls.go

```diff
@@ -28,16 +29,19 @@ type TLSConfiguration struct {
 
 // TLSDomain represents a domain (including wildcard domains) that is listed on a certificate's Subject Alternative Names (SAN) list.
 type TLSDomain struct {
-	ID   string `jsonapi:"primary,tls_domain"`
-	Type string `jsonapi:"attr,type"`
+	ID            string                  `jsonapi:"primary,tls_domain"`
+	Type          string                  `jsonapi:"attr,type"`
+	Activations   []*TLSActivation        `jsonapi:"relation,tls_activations,omitempty"`
+	Certificates  []*CustomTLSCertificate `jsonapi:"relation,tls_certificates,omitempty"`
+	Subscriptions []*TLSSubscription      `jsonapi:"relation,tls_subscriptions,omitempty"`
```

Comment:
I don't see `tls_subscription` in the [TLS domains data model](https://developer.fastly.com/reference/api/tls/custom-certs/domains/) 🤔

Reply:
True but it is in the example response!